.gitmodules

```diff
@@ -0,0 +1,4 @@
+[submodule "docs/.tmp/ansible"]
```

Comment:
Any reason you're doing this via a submodule rather than as a pip dependency? We do that in https://github.com/theforeman/obal/blob/master/requirements-docs.txt and is probably nicer

Reply:
At least the last time I looked, the docs/bin/ folder was not in the python distribution, and I need that to generate the docs.